.travis.yml

```diff
@@ -1,67 +1,64 @@
-  - JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl -n kube-system get pods -lcomponent=kube-addon-manager -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1;echo "waiting for kube-addon-manager to be available"; kubectl get pods --all-namespaces; done
-  - kubectl get deployment
-  - ./buildscripts/travis-build.sh
-  - sudo apt-get install -y open-iscsi
-  - sudo service iscsid start
-  - ./ci/ci-test.sh
+- kubectl cluster-info
+- JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}';
+  until kubectl -n kube-system get pods -lcomponent=kube-addon-manager -o jsonpath="$JSONPATH"
+  2>&1 | grep -q "Ready=True"; do sleep 1;echo "waiting for kube-addon-manager to
+  be available"; kubectl get pods --all-namespaces; done
+- kubectl get deployment
+- "./buildscripts/travis-build.sh"
+- sudo apt-get install -y open-iscsi
+- sudo service iscsid start
+- "./ci/ci-test.sh"
 after_success:
-  - make deploy-images
-  - bash <(curl -s https://codecov.io/bash)
+- make deploy-images
+- bash <(curl -s https://codecov.io/bash)
 notifications:
   email:
     recipients:
-    - kiran.mova@cloudbyte.com
+    - kiran.mova@openebs.io
 deploy:
   provider: releases
   api_key:
-    secure: na/NPsgDHGGRaWxRSCo5gH3TPrEutNvhEK3q2k99bbW2INe0FZ+FIPeuL9rqV8eCQi8SWJGHNFjFyMRR798RRSn8bdiK0pxJXzYvphUEH2Azzoqr65TaJHpHNTkv1WTK9OtgahT71MbmIx777U6Vd6ylyJyWja+LPhY/z66XOYQmuXR2ds7FRBlAcWg8C0KIFTLYlms5C9RKwLS2jP5C8tlJBQXMDEk7ejR1mKn3R6KQyyHICGKPGhNE+d7iMs0qhhuGIhcDwXl1olChAAITOGyWEmjc2GeUbFALo8OXdQx9qBO7saw75IzyYV/itBjE0RpuM90jKuFzKGiotSyw7Fs0KgrjHC7keuaNMqBWgKl6qoAj2a5VVEBtx8k941hRLs/VpjQ+K8wJJpjlSR8vh906b8e+HL8BKJEifF09fKBTLd0AWy9I3x6TolmRqiamvIHEkup1fZqblhhe2ZLvwuuyfl3t1FTkkon5BASgSqFdBAhR3eAD/LOtrghjaRX7wCZCzKDEaS9QLeu9UbC+bmnaOo60Gaeyp/DN5FLc4cV/vZozroesu+UEtQIrC6VDlFNYfY0V1ETKpfEQ4I8yByDHx/KjMWDyUGd8e5tm0qsD1lW1yVekh5CjQRHpzShkmKvFieeVfqVy/aGB4GrTeWSxcKiN8W0ekcgCRYut6y0=
-  file_glob: true
-  file:
-    - bin/csi/csi-linux_*.zip
+    secure: JYNHyi2m9pqOitOen9zX6FYjU0Q/uNjUhMpUDvHYq+RlqQ7zliJfiecRe91l+LKW1AmnGhNp4f2Shu2R9dQJwmle6y5ohsts3wRlRLAKxoex6jXo8uxCMVS5wE7+MlMmPWXaBSGy4Hcds+Jpzr90knO1avCDVdsM3ETbaE5tVQJuW8tgiCkWs+3V1xsjtpfT2s61tyikdbrRX1BevyN4/0mm0fJiowq8IelWwe+FEDUsVY3qg9D/aw7yEG9ApL34livin/VkWFqKq9ih1uB3timOjY3tVMyAgpmKcYqPxJPy3GjWYOeutwJcQZj8Zb4BowpVBj7rA70Fgqs9nVQXUyfc6LQ80+QIU+WBswqFGLA+SzyEGo/k+P45gYiawW4m3Bfap58X5hiQpUpHs5QBZuWC6cDt7q+BInf/+wlRI9pJF3QIv7KTEaySNdUtysMoAw0WGl5Sr7FtGrKKdyZB5E/eKbXGraCMXiY+jX2+YtUgyynO+G3gSY5PVr2BTseSFCCXLcvY8PG+x2ecc5nk4nNhpFkWrzGdwZJiriIMxrXStKB8dUWd55LEvuO475nbNUpnYVMY+br4NpUyqPyG33BM4my5sK8BimT6YO1tFsoFPx9TXSpuDAuk3QTzjZZET8P7SlFqJRwSjfLV1UkjrvHMkRaN7nSGmBaxtAP8Ucw=
+  file: bin/csi/csi-linux_*.zip
   skip_cleanup: true
   overwrite: true
   on:
     repo: openebs/csi
     tags: true
 branches:
   except:
-    - /^*-v[0-9]/
+  - "/^*-v[0-9]/"
```

Comment:
why this exception?

Reply:
was added longback due to some repush senarios issues in travis , not sure if its required anymore now.

Reply:
this is to check if tag is already exists don't push, if we restart the build again.